core/src/env/srml/types.rs

```diff
@@ -57,7 +63,7 @@ impl<'a> TryFrom<&'a [u8]> for AccountId {
 }
 
 /// The default SRML balance type.
-pub type Balance = u64;
+pub type Balance = u128;
```

Comment:
Is this the correct SRML `Balance` type? My latest knowledge was that it is still `u64`. I might be wrong though.

Reply:
Answer: Yes it is. Resolved.